charts/zookeeper/values.yaml

```diff
@@ -25,13 +25,24 @@ config: {}
   # tickTime: 2000
   # syncLimit: 2
   # quorumListenOnAllIPs: false
+storageType:
+    ## configure the storage backend type
+    ## accepted values : persistence/ephemeral
+    ## default option is
+    type: persistence
 
 persistence:
-  storageClassName: standard
-  ## specifying reclaim policy for PersistentVolumes
-  ## accepted values - Delete / Retain
-  reclaimPolicy: Delete
-  volumeSize: 20Gi
+    storageClassName: standard
+    # specifying reclaim policy for PersistentVolumes
+    # accepted values - Delete / Retain
+    reclaimPolicy: Delete
+    volumeSize: 20Gi
+
+ephemeral:
+    enabled: true
```

Comment:
please remove the `enabled` field from here. Whether ephemeral will be enabled/not will be determined by the value of `storage.type` field

Reply:
Done